gpii/configs/cloudBased.production.json

```diff
@@ -22,7 +22,7 @@
             }
         }
     },
-    "includes": [
+    "mergeConfigs": [
         "../node_modules/flatMatchMaker/configs/development.all.local.json",
```

Comment:
shouldn't we be using the "%flatMatchMaker" approach in these mergeConfigs definitions as well?